lib/koans/12_tuples.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Tuples do
+  use Koans
+
+  koan "tuples have a size" do
+    assert tuple_size({:a, :b, :c}) == :__
+  end
+
+  koan "tuples can contain different things" do
+    assert {:a, 1, "hi"} == :__
+  end
+
+  koan "you can pull out individual elements" do
+    assert elem({:a, "hi"}, 1) == :__
+  end
+
+  koan "you can change individual elements of a tuple" do
+    assert put_elem({:a, "hi"}, 1, "bye") == :__
+  end
+
+  koan "you can also simply extend a tuple with new stuff" do
+    assert Tuple.insert_at({:a, "hi"}, 1, :new_thing) == :__
+  end
+
+  koan "...avoid falling of the edge" do
+    assert_raise :__, fn ->  Tuple.insert_at({:a, "hi"}, 12, :new_thing) end
```

Comment:
Not sure about having a blank there.. How would the user know to fill this blank in? I guess if you have this open: http://elixir-lang.org/docs/stable/elixir/Tuple.html#insert_at/3. Are the koans supposed to be completed alongside the documentation?

I guess the overarching question is.. should I be able to figure out what the blank is by applying logic and reasoning or am I meant to swim in documentation as I'm doing these? I don't have an answer to it, just wanted to mention this.